File: libs/common/models/vw-pmp-psiprogram-by-category-list.ts



export class VwPmpPsiprogramByCategoryList {
    public cipSubSeriesCode: string = '';
    public cipSubSeries: string = '';
    public providerName: string = '';
    public logoUrl: string = '';
    public programName: string = '';
    public programId: number = 0;

    constructor(init? : Partial<VwPmpPsiprogramByCategoryList>) {
        
        Object.assign(this, init);
    }
}

File: apps/institution-api/Models/VwPmpPsiprogramByCategoryList.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class VwPmpPsiprogramByCategoryList
    {
        public string CipSubSeriesCode { get; set; }
        public string CipSubSeries { get; set; }
        public string ProviderName { get; set; }
        public string LogoUrl { get; set; }
        public string ProgramName { get; set; }

        public int ProgramId { get; set; }
  }
}
